src/parser.ts

```diff
@@ -57,20 +57,43 @@ export function exportSchedule(sheet: XLSX.WorkSheet) {
         .filter(row => meetingPatternsRegex.test(row['Meeting Patterns'] ?? ''))
         .filter(row => /^(Registered)|(Waitlisted)/.test(row['Registration Status'] ?? ''))
         .forEach(row => {
-            const [days, startTime, endTime, location] =
+            const [days, startTime, endTime, spot] =
                 meetingPatternsRegex.exec(row['Meeting Patterns'] ?? '')!.slice(1)
-            schedule.addEvent(
-                row['Course Listing']!,
-                `${row['Section']} with ${row['Instructor']}`,
-                location,
-                `${row['Start Date']} ${startTime}`,
-                `${row['Start Date']} ${endTime}`,
-                {
-                    freq: 'WEEKLY',
-                    until: `${row['End Date']} ${endTime}`,
-                    byday: convertDayOfWeekFormat(days),
-                },
-            )
+            const [startHour, startMinute] =
+                startTime.split(":")
+            const [endHour, endMinute] =
+                endTime.split(":")
+            const [month, day, badYear] =
+                row['Start Date']!.split("/")
+            const [endMonth, endDay, badEndYear] =
+                row['End Date']!.split("/")
+            // Workday for some ungodly reason puts the year as two digits.
+            const year = `20${badYear}`
+            const endYear = `20${badEndYear}`
+            // I'm sorry for this awful looking stuff, TypeScript is not my bread and butter.
```

Comment:
could probs do something like `[year, month, day, startHour, startMinute].map(parseInt)`, but doesn't matter that much

Reply:
I knew mapping was the answer I just wasn't sure how to do this in TS and it was 1am haha